packages/interface/src/components/preferences-modal-base-option/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { ToggleControl } from '@wordpress/components';
+
+function BaseOption( { help, label, isChecked, onChange, children } ) {
+	return (
+		<div className="interface-preferences-modal__option">
+			<ToggleControl
```

Comment:
Once https://github.com/WordPress/gutenberg/pull/39158 is merged, a next step will be to add a `PreferencesToggleControl` to the preferences package (similar to the way `PreferencesToggleMenuItem` works).

I'm not sure what the best way to integrate it with this generic toggle control will be 🤔 

The preferences package could use this component, but then that becomes only useful in the preferences modal (and I don't think `preferences` should depend on `interface`).

Reply:
The only reason to have this wrapper around `ToggleControl` is to allow children to be attached to it. If we implement a `PreferencesToggleControl` with that capacity, we don't need this component at all.